.github/workflows/build-and-push-Docker-image.yml

```diff
@@ -10,33 +12,41 @@ on:
     outputs:
       full-image-name:
         description: "Full pushed image name including host/registry, name, and tag"
-        value: ${{ jobs.docker-build.outputs.full-image-name }}
+        value: ${{ jobs.docker.outputs.full-image-name }}
+
 jobs:
-  docker-build:
+  docker:
     runs-on: ubuntu-latest
     permissions:
       packages: write
       contents: read
     timeout-minutes: 25
     outputs:
-      full-image-name: ${{ steps.push-image.outputs.full-image-name }}
+      full-image-name: ${{ steps.image-name.outputs.full-image-name }}
 
     steps:
-      - name: Checkout
-        # https://github.com/actions/checkout
-        uses: actions/checkout@v3
```

Comment:
again, seems really strange that you don't need to checkout the code in this repo, but I guess the official docker build action does that for you